tests/utils/test_display.py

```diff
@@ -0,0 +1,23 @@
+import pygfx as gfx
+import pytest
+from wgpu.gui.offscreen import WgpuCanvas
+
+
+def test_repeated_show():
+    pytest.xfail("Offscreen canvas can currently not be closed.")
+
+    cube = gfx.Mesh(
+        gfx.box_geometry(200, 200, 200),
+        gfx.MeshPhongMaterial(color="#336699"),
+    )
+
+    # always use offscreen canvas
+    # @almarklein do we have a fixture for that?
```

Comment:
There's this but it only works if the environment variable is set before importing the canvas from the auto namespace.

https://github.com/pygfx/pygfx/blob/2b762a314ed67ac81f3ed7ce1bbb2bdee2b6e0f5/examples/tests/test_examples.py#L59-L66